### PR TITLE
add reverting to snapshot after each test case

### PR DESCRIPTION
**Pull request by krzkaczor**
- localnode revert
- skeleton for new test
- new run cypress:dev task

### CODE REVIEW

File: cypress/integration/Balances.spec.js [hunk 1]
```diff
@@ -14,6 +13,6 @@ context('Balances', () => {
     cy.get(tid("wallets-continue")).contains("Continue").click();
 
     cy.get(tid("set-trade-from", tid("token-amount-value"))).contains(/8,999.... ETH/) // avoid asserting exact value here since it may easily change in future
-    cy.get(tid("set-trade-to", tid("token-amount-value"))).contains("1,000 DAI")
+    cy.get(tid("set-trade-to", tid("token-amount-value"))).contains("170 DAI")
```
Comment (peculiarity) [inline cypress/integration/Balances.spec.js:16]: Why did you change that to 170 ?

Also in those kind of tests probably we should get the balances and not use hardcoded values.

If we feed the account with other balance this test will fail.

File: cypress/integration/Order.spec.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/// <reference types="Cypress" />
+
+import { visitWithWeb3, tid, revertToSnapshot } from "../utils";
+
+context("Order", () => {
+  beforeEach(() => visitWithWeb3())
+  afterEach(() => revertToSnapshot());
+
+  it("should buy dai", () => {
```
Comment (peculiarity) [inline cypress/integration/Order.spec.js]: Just out of curiosity - what the reason behind this test?

File: cypress/utils/index.js [hunk 3]
```diff
@@ -1,19 +1,30 @@
 import Web3 from "web3";
 import PrivateKeyProvider from "truffle-privatekey-provider";
 
+// this is exactly the same as ganache.sh file in localnode
+const ACCOUNT_1_PRIV = "0x47be2b1589bb515b76b47c514be96b23cd60ee37e81d63c2ae9c92f7d7667e1a";
```
Comment (peculiarity) [inline cypress/utils/index.js:5]: Why do we have 1 and 3 but we don't have 2?

File: cypress/utils/index.js [hunk 4]
```diff
@@ -1,19 +1,30 @@
...
+const ACCOUNT_3_PRIV = "0x1ff8271bf14ac9bef0b641cced40dc2a7ebd2e37d8e16d25b4aa1911364219af";
+
+export let web3;
+export let lastSnapshotId;
+
 export function visitWithWeb3(path = "") {
-  const provider = new PrivateKeyProvider(Cypress.env("ETH_PRIV_KEY").slice(2), Cypress.env("ETH_PROVIDER"));
-  const web3 = new Web3(provider);
+  const provider = new PrivateKeyProvider(ACCOUNT_3_PRIV.slice(2), Cypress.env("ETH_PROVIDER"));
```
Comment (peculiarity) [inline cypress/utils/index.js]: I would go with `ACCOUNT_3_PRIV.replace("0x","")` 
It's more visible what you are trying to do.

File: cypress/utils/index.js [hunk 5]
```diff
@@ -1,19 +1,30 @@
...
+  web3 = new Web3(provider);
 
-  cy.then(() => saveBlockchain(web3)()).debug();
+  return cy.then(() => saveBlockchain(web3)()).then(r => {
```
Comment (peculiarity) [inline cypress/utils/index.js:15]: Can we give some meaning to the result? 
I'm little bit confused what kind of number it will return?

File: cypress/utils/index.js (hunk 3 above)
Comment (krzkaczor) [reply cypress/utils/index.js:5]: it's corresponding to list hardcoded in localnode/ganache.sh I used account3 here since it was least used in the past 🤷‍♂️

File: cypress/utils/index.js (hunk 5 above)
Comment (krzkaczor) [reply cypress/utils/index.js:15]: that's why we should start using TS :D it's JSON RPC response defined somewhere in web3js docs. Meaning is provided on the very next line `lastSnapshotId`

File: cypress/integration/Order.spec.js (hunk 2 above)
Comment (krzkaczor) [reply cypress/integration/Order.spec.js]: it was supposed to be just a skeleton for next test, i will remove it since its not doing anything and i think you did some work on other branch

Comment (peculiarity) [reply cypress/integration/Order.spec.js]: Yeah, I almost had a whole test regarding selling ETH but it fails with the error I gave you earlier  - `VM execution error - no revert` or something like this

**Review by peculiarity — APPROVED** [review]
